src/main/java/frc/robot/Robot.java

```diff
@@ -32,17 +45,29 @@
   private final SubsystemManager mSubsystemManager = SubsystemManager.getInstance();
 
   //Subsystems
-  private final Drive mDrive = Drive.getInstance();
   private final VisionManager mVisionManager = VisionManager.getInstance();
   private final Shooter mShooter = Shooter.getInstance();
 
   //Variables from State
 
+  
+  static Potentiometer mPot;
```

Comment:
Member variables aren't static. Either update the name to reflect static, or change to a non-static.

Reply:
Fixed